src/main/java/dao/RecordDaoImpl.java

```diff
@@ -0,0 +1,30 @@
+package dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class RecordDaoImpl implements RecordDao {
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> result;
+        try {
+            result = Files.readAllLines(Path.of(fileName));
```

Comment:
```suggestion
            return Files.readAllLines(Path.of(fileName));
```

Reply:
Didn`t quite understand why the interface should be removed. + this interface showed it in the lesson as well. Could you give more details, please, for my understand?